### DISCUSSION TITLE
where is node_modules/zigbee-herdsman-converters

### DISCUSSION THREAD

**Discussion by ThierryBourbon** [Koenkk/zigbee2mqtt#12248] [General]
Newbe question
how to reach 
node_modules/zigbee-herdsman-converters
in my HA supervised?

**Comment by lux73** [comment]
login to ssh:

`find / -name zigbee-herdsman-converters`

**Reply by ThierryBourbon** [reply]
Works. Thank's

**Reply by lux73** [reply]
you're welcome :-)

could please post the correct Path - it might be useful if others find this Thread

on my vanilla z2m System is:

`/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters`

**Reply by ThierryBourbon** [reply]
I have a special system (home assistant on open mediavault) so the the path is complicated :-)
```
/var/lib/docker/overlay2/90af22c27873e0ec89a69b29909103a23e8e178a7c31c930925aab66c7cbf5d4/diff/app/node_modules/zigbee-herdsman-converters
```
